CMakeLists.txt

```diff
@@ -155,6 +167,26 @@ if (NOT "${NSYNC_LANGUAGE}X" STREQUAL "c++11X")
 	elseif ("${CMAKE_SYSTEM_NAME}X" STREQUAL "OpenBSDX")
 		include_directories ("${PROJECT_SOURCE_DIR}/platform/openbsd")
 		set (NSYNC_POSIX ON)
+      elseif ( "${CMAKE_SYSTEM_NAME}X" STREQUAL "AndroidX")
+
+        message("ANDROID!!!")
+        
+      if ("${ANDROID_ARCH}X" STREQUAL "armv7-aX")
```

Comment:
There is no `ANROID_ARCH` variable. Available variables to analyze:
* https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH.html
* https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH_ABI.html

Reply:
Done.